extensions/exposition/components/identity.federation/manifest.toa.yaml

```diff
@@ -2,42 +2,59 @@ namespace: identity
 name: federation
 
 entity:
+  associated: true
   schema:
     type: object
     properties:
+      authority:
+        description: The authority token is valid for
+        type: string
```

Comment:
Wouldn't be more logical then to just have `aud` here?

Reply:
Actually, I had the exact same idea when doing this. My arguments for naming it `authority` here were:

1. It is the same piece of information as in other entities/contracts, and having multiple names for the same information is almost always misleading.
2. A significant source of confusion, specifically in this entity, would be that the neighbor properties `iss` and `sub` are the values that are carried by the token, while the token's `aud` is verified and not stored. Having `aud` here may lead to the idea that it is an `aud` from the token.